docs-gen/content/docs/examples/pipeline_dependencies.md

```diff
@@ -12,7 +12,7 @@ In this example, we break up the penguin example pipeline into two pipelines:
 - The *Penguin Examples Pipeline* has a single pipeline step that imports the CSV example and outputs it as an artifact
 - The *Penguin Training Pipeline* references the previously produces example and trains the model
 
-![ensembling.svg](/images/ensembling.svg)
+![ensembling.svg]({{< param "subpath" >}}/images/ensembling.svg)
```

Comment:
we have to do this for images now, as we're no longer setting `canonifyURLs` to true, so all links are relative. Without using the subpath param (defined in `hugo.toml`), this would try to get the image from `https://sky-uk.github.io/images/cd-ct.svg` rather than `https://sky-uk.github.io/kfp-operator/images/cd-ct.svg`